src/elements/content-uploader/ContentUploader.js

```diff
@@ -89,7 +89,7 @@ type State = {
     view: View,
 };
 
-const CHUNKED_UPLOAD_MIN_SIZE_BYTES = 52428800; // 50MB
+const CHUNKED_UPLOAD_MIN_SIZE_BYTES = 104857600; // 100MB
```

Comment:
Could this be moved to a constants file?

Reply:
Outside the scope of this PR